sktime/markov/msm/augmented_msm.py

```diff
@@ -0,0 +1,623 @@
+# This file is part of PyEMMA.
+#
+# Copyright (c) 2014-2019 Computational Molecular Biology Group, Freie Universitaet Berlin (GER)
+#
+# PyEMMA is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+__author__ = 'Simon Olsson'
+
+import logging
+from typing import Optional
+
+import numpy as np
+from msmtools import estimation as msmest
```

Comment:
not a criticism just a question: we are still using/depending on `msmtools`?

Reply:
msmtools is not just developed by us but also by someone from the ZIB, we have to discuss how to proceed with this. For now we depend on it although it would be better for this project to have it included, I agree. And then there is the time factor of ourse :)

Reply:
got it just wanted to make sure because i was confused about there being msm estimators coming from both msmtools and internally

Reply:
msmtools offers a functional interface and the optimization methods, in here we just import and repackage them